.github/workflows/ci.yml

```diff
@@ -185,21 +190,15 @@ jobs:
       with:
         version: 1.9
 
-    - name: Cache
-      uses: actions/cache@v3
-      with:
-        path: |
-          ~/.cache
-          .tox/
```

Comment:
Unless explicitly incompatible with `julia-actions/cache`, continue caching `.tox/` and `~/.cache/pip/`

Reply:
well Julia_actions/cache does not support the path argument

Reply:
I meant continue caching those with `actions/cache`